Makefile

```
@@ -40,7 +40,7 @@ goodbye:
 # Local #
 #-------#
 dev: hello build init_backend _up import_sample_data create_mongodb_indexes refresh_product_tags
-	@echo "🥫 You should be able to access your local install of Open Food Facts at http://productopener.localhost"
+	@echo "🥫 You should be able to access your local install of Open Food Facts at http://world.openfoodfacts.localhost//"
```

```suggestion
	@echo "🥫 You should be able to access your local install of Open Food Facts at http://world.openfoodfacts.localhost/"
```

Only 1 trailing / is needed: http://world.openfoodfacts.localhost/

---

@stephanegigandet  done